pynsee/localdata/_get_geo_list_simple.py

```diff
@@ -12,9 +12,10 @@
 
 
 @lru_cache(maxsize=None)
-def _get_geo_list_simple(geo, progress_bar=False):
-
+def _get_geo_list_simple(geo, date=None, progress_bar=False):

```

Comment:
shouldn't this be in a separate PR? this one is already quite big (for intrinsic reasons) and this change is drowned in unrelated changes.